plugins/goodreads.koplugin/main.lua

```diff
@@ -6,6 +6,7 @@ local InfoMessage = require("ui/widget/infomessage")
 local UIManager = require("ui/uimanager")
 local Screen = require("device").screen
 local _ = require("gettext")
+local NetworkMgr = require("ui/network/manager")
```

Comment:
@houqp does it make a difference whether this require is here or within `local Goodreads`?